tests/plugins/test_koji_import.py

```diff
@@ -1841,3 +1842,62 @@ def test_koji_import_primary_images(self, tmpdir, os_env, add_tag_conf_primaries
             return
 
         runner.run()
+
+    @pytest.mark.parametrize('resolve_run', [
+        True,
+        False,
```

Comment:
This works fine, but it's not very efficient because it ends up trying all of the 'comp' values, for instance, even when the plugin is not run in the scenario.

Reply:
made another test for that, so it is more efficient